mu-plugins/blocks/sidebar-container/postcss/style.pcss

```diff
@@ -14,41 +14,59 @@
 @media (min-width: 1200px) {
 	.wp-block-wporg-sidebar-container {
 		--local--block-end-sidebar--width: 340px;
+		--local--top-offset: calc(var(--wp-admin--admin-bar--height, 0px) + var(--wp--custom--local-navigation-bar--spacing--height, 60px));
 
-		position: absolute;
-		top: calc(var(--wp-global-header-offset, 90px) + var(--wp--custom--local-navigation-bar--spacing--height, 60px));
-
-		/* Right offset should be "edge spacing" at minimum, otherwise calculate it to be centered. */
-		right: max(var(--wp--preset--spacing--edge-space), calc((100% - var(--wp--style--global--wide-size)) / 2));
 		width: var(--local--block-end-sidebar--width);
-		margin-top: var(--wp--custom--wporg-sidebar-container--spacing--margin--top);
 		margin-bottom: 0 !important;
+		padding: var(--wp--preset--spacing--20) 0;
+
+		main & {
```

Comment:
This seems to be the cause of [this issue for me](https://github.com/WordPress/wporg-developer/pull/469#issuecomment-1894672967)— this isn't sass, so the nesting rules aren't the same. This seems to be ignored by postcss, and then stripped out by cssnano, so the rules below are applied to all `.wp-block-wporg-sidebar-container`s. I don't know why it works for you & @renintw though.

If I move this code to a new section `main .wp-block-wporg-sidebar-container {…` it works as expected.

Reply:
> I don't know why it works for you & @renintw though.

Nice catch! All this seems to work with a dev build, but it all breaks with a prod build. Starting to think moving to one CSS style would be a really good idea as switching feature sets contributes to this sort of issue. I think that was suggested to be postcss.

Reply:
> I think that was suggested to be postcss.

I think that was before wp-scripts natively supported sass… and maybe if we switched to sass here, we could remove the custom build script here.